File: easy/1295/1295.js
/**
 * Runtime: 80 ms, faster than 75.32%
 * Memory Usage: 39.1 MB, less than 77.61%
 */
var findNumbers = function(nums) {
  let result = 0;

  nums.forEach(e => {
    if (e.toString().length % 2 == 0) ++result;
  });

  return result;
};

File: easy/1295/1295.rb
# Runtime: 48 ms, faster than 95.38%
# Memory Usage: 210.2 MB, less than 65.38%

def find_numbers(nums)
  result =  0

  for x in nums do
    if x.to_s.length % 2 == 0
      result += 1
    end
  end

  result
end
